influxdb3_write/src/write_buffer/mod.rs

```diff
@@ -342,6 +326,42 @@ impl<W: Wal, T: TimeProvider, P: Persister> WriteBufferImpl<W, T, P> {
         Ok(chunks)
     }
 
+    pub async fn cache_parquet(
+        &self,
+        db_name: &str,
+        table_name: &str,
+        records: SendableRecordBatchStream,
+    ) -> Result<(), Error> {
+        // TODO get these from the buffer somehow
+        let min_time = 0;
+        let max_time = 0;
```

Comment:
Can these be determined from the buffer, or would the caller providing the `RecordBatch` stream have to provide them, i.e., as arguments to the function?

Reply:
Yeah, definitely don't want to have to extract it from the stream, hopefully they are known a priori.

Reply:
The buffer keeps the min and max time so it'll have it handy when it calls to this function.